blockchains/erc20/lib/util.js

```diff
@@ -18,7 +20,18 @@ function stableSort(array, sortFunc) {
   array.forEach(x => delete x._position);
 }
 
+async function readJsonFile(filePath) {
+  try {
+    const data = await fs.readFile(filePath, 'utf8');
+    const parsedContracts = JSON.parse(data);
+    return parsedContracts;
+  } catch (err) {
+    console.error(err);
```

Comment:
Why do we not throw something? Also, it'd be better if we sticked with logger I think